client/components/CommentsSection.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+
+import { Col, Row } from "reactstrap";
+
+import {
+  postComment,
+  postCommentThread,
+  getComments
+} from "../utils/apiWrapper";
+
+import Comment from "./Comment";
+import CommentEditor from "./CommentEditor";
+
+export default function CommentsSection(props) {
+  const { projectId } = props;
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    fetchComments();
+  }, []);
+
+  const renderComments = comments => {
+    let commentList = [];
+    for (let [index, comment] of comments.entries()) {
+      commentList.push(
+        <Row>
+          <Comment
+            comment={comment}
+            postThread={content => postThread(content, index)}
+          />
+        </Row>
+      );
+    }
+    return commentList;
+  };
+
+  const fetchComments = () => {
+    getComments(projectId).then(data => {
+      const { comments } = data.data;
```

Comment:
This is a bit nitpicky, but could we be more explicit with naming the parent here such that it doesnt have the same name `data` as the child?